src/theme/components/accordion.ts

```diff
@@ -1,21 +1,28 @@
 const containerStyles = {
   borderLeftWidth: "1px",
   borderRightWidth: "1px",
-  borderColor: "ui.gray.medium",
   width: "100%",
+  _hover: {
```

Comment:
@bigfishdesign13, check out my earlier comments on the border color. Maybe the default border color should be something different? Right now, if I set the hover border to `ui.gray.medium` there is no change because the default is already `ui.gray.medium`.